EDA Feature Engineering Machine Learning on tabular data from my end

ancarlosmendes opened the pull request.

farsim-hossain committed:
feature eng on tabular data from my end

farsim-hossain committed:
basid EDA with PBI

ancarlosmendes committed:
eda-features-engineering-ml

EDA Features Engineering Machine Learning

ancarlosmendes committed:
eda-features-engineering-model

EDA Features Engineering Machine Learning

leokook committed:
Add files via upload

leokook committed:
Create Notebooks

leokook committed:
Create Class_separability

leokook committed:
Add files via upload

leokook committed:
Delete Notebooks./Class_separability

leokook committed:
Delete Classes_separability.ipynb

leokook committed:
Delete Notebooks

leokook committed:
Add files via upload

wyieborn committed:
Baseline modeling using numerical and categorical features with RF

wyieborn committed:
Delete Notebooks. directory

Fixing Pull issue due to Folder Name.

wyieborn committed:
Merge branch 'develop' of https://github.com/wyieborn/Pet-Finder-Adoption into develop

wyieborn committed:
Model save path handling

wyieborn committed:
refactoring and renaming methods

leokook committed:
Create Class

leokook committed:
Add files via upload

leokook committed:
Delete Notebooks/Class

ancarlosmendes committed:
new features using data transformation

New features have created using data transformation

wyieborn committed:
Flask API; New Model With 90% acc; StreamLit Prototype; ImgFeat Integration; Refactoring gdscv;

wyieborn committed:
Flask api reload fix; img feat path conf edits

wyieborn committed:
Update README.md

Conifguration guide and steps to run the application.

wyieborn committed:
Update README.md

spelling correction

wyieborn committed:
Update README.md

wyieborn committed:
Update README.md

clearifications for image path

wyieborn committed:
removed unused path

wyieborn committed:
Merge branch 'develop' of https://github.com/wyieborn/Pet-Finder-Adoption into develop

wyieborn committed:
requirements.txt for installing dependencies

wyieborn committed:
Update README.md

steps added for requirements.txt

wyieborn committed:
Update README.md

spelling correction

wyieborn committed:
Update README.md

farsim-hossain committed:
organizations_dataset

ancarlosmendes committed:
Create final-notebook.ipynb

Final Notebook with Feature Engineering , Extract Features from Images, Sentiment Analysis, EDA, and Classes_separability

wyieborn committed:
Update .gitignore

venv omit rule

wyieborn committed:
added part 2 model

farsim-hossain reviewed (commented):
Works well in the demo

wyieborn committed:
changes to part2; final notebook

leokook committed:
Add files via upload

After V7 on Dec/05 the Final Notebook gets an update.

leokook committed:
Delete FinalNotebook/Final - Notebook.html

leokook committed:
Add files via upload

As final model was uploaded on late night of Dec/04 the Final Notebook was also updated.

leokook committed:
Delete FinalNotebook/Final - Notebook.html

leokook committed:
Add files via upload

leokook committed:
Add files via upload

felipefe20 committed:
adding notebooks mlflow and shap

leokook committed:
Add files via upload
